notebooks/model_conversion-classification.ipynb

```diff
@@ -0,0 +1,656 @@
+     "exception": false,
+     "start_time": "2023-04-03T11:44:07.971506",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "# Convert the models to torchscript using Hummingbird and test the results (visualized further below)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f811271",
+   "metadata": {
+    "papermill": {
+     "duration": 0.007784,
+     "end_time": "2023-04-03T11:44:07.996113",
+     "exception": false,
+     "start_time": "2023-04-03T11:44:07.988329",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "RandomForest and LinearSVC"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "1ec286fa",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-04-03T11:44:08.014735Z",
+     "iopub.status.busy": "2023-04-03T11:44:08.013821Z",
+     "iopub.status.idle": "2023-04-03T11:44:08.463774Z",
+     "shell.execute_reply": "2023-04-03T11:44:08.462727Z"
+    },
+    "papermill": {
+     "duration": 0.461911,
+     "end_time": "2023-04-03T11:44:08.466460",
+     "exception": false,
+     "start_time": "2023-04-03T11:44:08.004549",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/opt/conda/lib/python3.7/site-packages/sklearn/utils/deprecation.py:103: FutureWarning: Attribute `n_features_` was deprecated in version 1.0 and will be removed in 1.2. Use `n_features_in_` instead.\n",
+      "  warnings.warn(msg, category=FutureWarning)\n"
+     ]
+    }
+   ],
+   "source": [
+    "torch_models = []\n",
+    "for model in sklearn_models[:2]:\n",
+    "    torch_model = hb.convert(model, \"TorchScript\", \n",
+    "                             test_input = np.array(X_train[0]).reshape(1, -1))\n",
+    "    torch_models.append(torch_model)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1c798448",
+   "metadata": {
+    "papermill": {
+     "duration": 0.008323,
+     "end_time": "2023-04-03T11:44:08.483954",
+     "exception": false,
+     "start_time": "2023-04-03T11:44:08.475631",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "source": [
+    "K Nearest Neighbors"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "836880dc",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2023-04-03T11:44:08.502471Z",
+     "iopub.status.busy": "2023-04-03T11:44:08.501534Z",
+     "iopub.status.idle": "2023-04-03T11:44:08.553885Z",
+     "shell.execute_reply": "2023-04-03T11:44:08.552831Z"
+    },
+    "papermill": {
+     "duration": 0.064338,
+     "end_time": "2023-04-03T11:44:08.556536",
+     "exception": false,
+     "start_time": "2023-04-03T11:44:08.492198",
+     "status": "completed"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "extra_config = {hb.operator_converters.constants.BATCH_SIZE: 1}\n",
```

Comment:
could you explain why this is necessary for the knn classifier?

Reply:
Otherwise, the following exception is raised:

`RuntimeError: Hummingbird requires explicit specification of batch_size parameter when compiling KNeighborsClassifier`